common/extensions/brave_pdf_download_browsertest.cc

```diff
@@ -131,6 +134,22 @@ class BravePDFDownloadTest : public extensions::ExtensionFunctionalTest {
 public:
   BravePDFDownloadTest() = default;
   ~BravePDFDownloadTest() override = default;
+  void SetUpOnMainThread() override {
+    extensions::ExtensionFunctionalTest::SetUpOnMainThread();
+    // set up extension blacklist for testing
+    g_brave_browser_process->extension_whitelist_service()
+      ->extension_whitelist_client_
+      ->addToBlacklist("mhjfbmdgcfjbbpaeojofohoefgiehjai");
+    ExtensionRegistry* registry =
```

Comment:
Please advise whether this should be done by the extension whitelist service itself, or if doing this manually for the test is okay.

Reply:
You can remove this after you see my comments above.